modules/dfem_diffusion/dfem_diffusion_solver.h

```diff
@@ -81,16 +70,38 @@ class Solver : public opensn::Solver
                       size_t ccfi,
                       double epsilon = 1.0e-12);
 
-  /**
-   * Updates the field functions with the latest data.
-   */
-  void UpdateFieldFunctions();
+  typedef std::pair<opensn::diffusion::BoundaryType, std::vector<double>> BoundaryInfo;
+  typedef std::map<std::string, BoundaryInfo> BoundaryPreferences;
+
+  std::shared_ptr<MeshContinuum> grid_ptr_ = nullptr;
+
+  std::shared_ptr<SpatialDiscretization> sdm_ptr_ = nullptr;
+
+  size_t num_local_dofs_ = 0;
+  size_t num_globl_dofs_ = 0;
+
+  std::vector<double> field_;
+
+  /// approx solution
+  Vec x_ = nullptr;
+  /// RHS
+  Vec b_ = nullptr;
+  /// linear system matrix
+  Mat A_ = nullptr;
```

Comment:
For short member declarations, does it make more sense to put descriptions to the right?

Reply:
I usually do not do that. One has to remember to use `///<` otherwise the doco ends up with the wrong member variable. And if the text gets longer (for different reasons), clang starts to format into a column over multiple lines while when I use the comment before, I get the full width.  So, I personally do not mix the `///` and `///<` comments.